src/main/java/com/cs56fitnessapp/models/StrengthTraining.java

```diff
@@ -0,0 +1,54 @@
+package com.cs56fitnessapp.models;
+
+import  com.cs56fitnessapp.models.workout.Workout;
+import java.util.ArrayList;
+import java.time.LocalDate;
+
+/**
+ *  @author Daniel Cervantes
+ *  @author Ksenia Koldaeva
+ *
+ * Date created: 10/18/17
+ * Last updated: 10/18/17
+ */
+
+
+public class StrengthTraining extends Workout {
+
+
+    private ArrayList<String> exerciseList = new ArrayList<>(){{
+
+        add("Bench Press\n");
+        add("Bicep Curls\n");
+        add("Shoulder Press\n");
+        add("Lateral Raises\n");
+        add("Tricep Extensions\n");
+        add("Bent Over Rows\n");
+        add("Sit Ups\n");
+        add("Back Squats\n");
+        add("Front Squats\n");
+        add("Deadlift\n");
+        add("Leg Press\n");
+        add("Lunges\n");
+        add("Kettle Bell Swings\n");
+        add("Leg Lifts\n");
+
+    }};
+
+
+    public StrengthTraining(User user, LocalDate date) {
+        super(user, date);
+        this.exerciseList = new ArrayList<>();
+    }
+
+
+    @Override
+    public int getCaloriesOut() {
```

Comment:
All overridden methods must have actual implementations.
E.g. I completed my strength workout composed of  20 x LateralRaise + 20 x Lunge + 50 x BackSquat, and I come to you (since you keep track about it in your StrengthTraining class), and I ask - how many calories have I burned? 

This is what getCaloriesOut() should return.
Currently, no matter what exercises I have completed, your method returns 0 calories burned. I bet I would be frustrated.

Same goes for getActiveTimeMins().